README.md

```diff
@@ -1,2 +1,18 @@
 # Hactoberfest-Hack-Day-2019
 An event organised by GNU/Linux Users' Group, NIT Durgapur for the promotion of hacktoberfest 2019.
+
+# What is Hacktoberfest?
+Hacktoberfest is a month long virtual festival event to celebrate open source contributions presented by Digital Ocean and DEV. It is the easiest way to get into open source!
+During the entire month of October 2019, all you have to do is contribute to any open source projects and open at least 4 pull requests. Yes, any project and any kind of contributions. You don’t need to be an expert in programming or coding. It can be a be a bug fix, improvement, or even a documentation change! And win an awesome T-shirt and awesome stickers.
+# Why Should I Contribute?
+Support open source and earn a limited edition T-shirt!
+
+But, it’s not just about the t-shirts or stickers. Its about supporting open source. Its about celebrating open source, and giving it back. If you’ve never contributed to open source before, this is the perfect time to get started because Hacktoberfest provides a large list of available contribution opportunities (and yes, there are always plenty for beginners too).
+#  What Can I Contribute?
+Hacktoberfest® is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge.
+It doesn’t need to be code. It can be even documentation. Or spelling fix.
+You can contribute to any open source project hosted on Github.com and contribute anything between October 1 to October 31 midnight in 2019. You will find plenty of issues labeled with hacktoberfest or good-first-issue etc on Github. These will be the simple ones most probably and easy to fix.
+
+# How Can I Contribute?
+First you need to get yourself registered for Hacktoberfest. Signup at the Hacktoberfest 2019 website below.
+Then as per your interest in any domain or language you can cotribute to any of the organisation.
```

Comment:
Remove this section. Add 
```
First you need to get yourself registered for Hacktoberfest. Signup at the Hacktoberfest 2019 website below.
```

 as the first point in `Contribution Guidlines` section.

Reply:
Also, Can you please rebase the PR ?